src/components/ui/TopicsCard.tsx

```diff
@@ -22,10 +23,8 @@ function TopicsCard({
 }: ITopicsCard) {
   const { mutate } = useMutation({
     mutationFn: getBookMarkByTopic,
-    onSuccess: () => {
-      console.log("Bookmarks fetched successfully");
-    },
     onSettled: (data) => {
+      // ISSUE: need to change the logic for this because we want to bookmark at top which is recently created
```

Comment:
ISSUE: need to change the logic for this because we want to bookmark at top which is recently created